build/Targets/Imports.targets

```diff
@@ -155,6 +155,7 @@
           <PropertyGroup>
             <AssemblyOriginatorKeyFile>$(MSBuildThisFileDirectory)..\Strong Name Keys\RoslynInternalKey.Private.snk</AssemblyOriginatorKeyFile>
             <DelaySign>false</DelaySign>
+            <PublicSign>true</PublicSign>
```

Comment:
This feels wrong.  This PropertyGroup is only hit when we are fully signing binaries, not public signing.